src/Microsoft.EntityFrameworkCore.UnitOfWork/IRepository.cs

```diff
@@ -42,15 +43,17 @@ public interface IRepository<TEntity> where TEntity : class
         /// <param name="pageIndex">The index of page.</param>
         /// <param name="pageSize">The size of the page.</param>
         /// <param name="disableTracking"><c>True</c> to disable changing tracking; otherwise, <c>false</c>. Default to <c>true</c>.</param>
+        /// <param name="include">A function to include navigation properties</param>
         /// <returns>An <see cref="IPagedList{TEntity}"/> that contains elements that satisfy the condition specified by <paramref name="predicate"/>.</returns>
         /// <remarks>This method default no-tracking query.</remarks>
-        IPagedList<TEntity> GetPagedList(Expression<Func<TEntity, bool>> predicate = null, Func<IQueryable<TEntity>, IOrderedQueryable<TEntity>> orderBy = null, int pageIndex = 0, int pageSize = 20, bool disableTracking = true);
+        IPagedList<TEntity> GetPagedList(Expression<Func<TEntity, bool>> predicate = null, Func<IQueryable<TEntity>, IOrderedQueryable<TEntity>> orderBy = null, int pageIndex = 0, int pageSize = 20, bool disableTracking = true, Func<IQueryable<TEntity>, IIncludableQueryable<TEntity, object>> include = null);
```

Comment:
In case when parameter `include` is after `orderBy` will work code `GetPagedList(predicate)`, `GetPagedList(predicate, order)` and calls with named parameters. For `GetPagedList(x, y, 1)`, `GetPagedList(x, y, 0, 30)` and `GetPagedList(x, y, 0, 30, fase)`  is breaking change.
so, If we do not want to break the user code, the only option is to place `include` at the end of parameters.

PS: I do not currently use this method, and i can place `include` after `orderBy`.